src/sidebar/components/group-list.js

```diff
@@ -3,6 +3,8 @@
 var { isThirdPartyUser } = require('../util/account-id');
 var isThirdPartyService = require('../util/is-third-party-service');
 var serviceConfig = require('../service-config');
+const memoize = require('../util/memoize');
+const sortedGroups = memoize(require('../util/group-organizations'));
```

Comment:
It would slightly help grep-ability here if the function name here had the same name as in the module where it is defined (`groupOrganizations`).

Reply:
Good feedback, thanks!